libbeat/outputs/kafka/kafka_integration_test.go

```diff
@@ -47,6 +47,21 @@ const (
 	kafkaDefaultPort = "9092"
 )
 
+var (
+	kafkaHost = fmt.Sprintf("%v:%v",
+		getenv("KAFKA_HOST", kafkaDefaultHost),
+		getenv("KAFKA_PORT", kafkaDefaultPort),
+	)
+	kafkaKerberosHost = fmt.Sprintf("%v:%v",
+		getenv("KAFKA_KERBEROS_HOST", kafkaDefaultHost),
+		getenv("KAFKA_PORT", kafkaDefaultPort),
```

Comment:
Kafka secure listener is on port `9093`, how does it work? :thinking:

Reply:
I configured `9092` in an override when running Kafka in the `run.sh`. Or I am not sure I understand your question.

Reply:
I refer to this, is Kerberos authentication required in both plaintext and SASL listeners?
```
    --override listeners=PLAINTEXT://kafka_kerberos:9092,SASL_SSL://kafka_kerberos:9093 \
    --override advertised.listeners=PLAINTEXT://${ADVERTISED_HOST}:9092,SASL_SSL://${ADVERTISED_HOST}:9093 \
```